faq/faq.rst

```diff
@@ -109,23 +109,26 @@ Certain commands, when run on the coordinator node, do not get propagated to the
 
 * ``CREATE ROLE/USER``
 * ``CREATE FUNCTION``
```

Comment:
I am in doubt if we should keep this here. We don't have tables in here either.
Any strong reasons to keep `CREATE FUNCTION` in this list?